### ISSUE TITLE
Branch name mono-repos not expanded properly

### ISSUE THREAD

**Issue by bcoe** [googleapis/release-please#1135]
The branch name does not replace `${branch}`, and becomes `chore: release ${branch}`.

Refs: https://github.com/istanbuljs/istanbuljs/pull/654

**Comment by bcoe** [comment]
Promoting to `p1`, because this breaks tagging:

```
```bash
⚠ pullRequestTitlePattern miss the part of '${version}'
✖ Bad pull request title: 'chore: release ${branch}'
⚠ pullRequestTitlePattern miss the part of '${scope}'
⚠ pullRequestTitlePattern miss the part of '${component}'
⚠ pullRequestTitlePattern miss the part of '${version}'
✖ Bad pull request title: 'chore: release ${branch}'
```

**Comment by chingor13** [comment]
This should have been fixed in #1129: https://github.com/googleapis/release-please/pull/1129/files#diff-8fde470d821a54dc84fa30c40cd642499a533ddb5a836a56fd64ddf91c4c929eR154
